test/Microsoft.ML.Tests/Transformers/NormalizerTests.cs

```diff
@@ -207,6 +208,144 @@ public void NormalizerParameters()
+        {
+            string dataPath = GetDataPath("iris.txt");
+            var context = new MLContext(seed: 0);
+
+            var loader = new TextLoader(context, new TextLoader.Options
+            {
+                Columns = new[] {
+                    new TextLoader.Column("Label", DataKind.Single, 0),
+                    new TextLoader.Column("float1", DataKind.Single, 1),
+                    new TextLoader.Column("float4", DataKind.Single, new[]{new TextLoader.Range(1, 4) }),
+                    new TextLoader.Column("double1", DataKind.Double, 1),
+                    new TextLoader.Column("double4", DataKind.Double, new[]{new TextLoader.Range(1, 4) }),
+                    new TextLoader.Column("int1", DataKind.Int32, 0),
+                    new TextLoader.Column("float0", DataKind.Single, new[]{ new TextLoader.Range { Min = 1, VariableEnd = true } })
+                },
+                HasHeader = true
+            }, new MultiFileSource(dataPath));
+
+            var est = context.Transforms.NormalizeMinMax(
+                new[] { new InputOutputColumnPair("float1"), new InputOutputColumnPair("float4"),
+                    new InputOutputColumnPair("double1"), new InputOutputColumnPair("double4"), })
+                    .Append(context.Transforms.NormalizeBinning(
+                                new[] {new InputOutputColumnPair("float1bin", "float1"), new InputOutputColumnPair("float4bin", "float4"),
+                                    new InputOutputColumnPair("double1bin", "double1"), new InputOutputColumnPair("double4bin", "double4")}))
+                    .Append(context.Transforms.NormalizeMeanVariance(
+                                new[] {new InputOutputColumnPair("float1mv", "float1"), new InputOutputColumnPair("float4mv", "float4"),
+                                    new InputOutputColumnPair("double1mv", "double1"), new InputOutputColumnPair("double4mv", "double4")}))
+                    .Append(context.Transforms.NormalizeLogMeanVariance(
+                                new[] {new InputOutputColumnPair("float1lmv", "float1"), new InputOutputColumnPair("float4lmv", "float4"),
+                                    new InputOutputColumnPair("double1lmv", "double1"), new InputOutputColumnPair("double4lmv", "double4")}))
+                    .Append(context.Transforms.NormalizeSupervisedBinning(
+                                new[] {new InputOutputColumnPair("float1nsb", "float1"), new InputOutputColumnPair("float4nsb", "float4"),
+                                    new InputOutputColumnPair("double1nsb", "double1"), new InputOutputColumnPair("double4nsb", "double4")}));
+
+            var data = loader.Load(dataPath);
+
+            var transformer = est.Fit(data);
+            var transformers = transformer.ToImmutableArray();
+            var floatAffineData = ((NormalizingTransformer)transformers[0]).Columns[0].ModelParameters as NormalizingTransformer.AffineNormalizerModelParameters<float>;
```

Comment:
```suggestion
            var floatAffineModel = ((NormalizingTransformer)transformers[0]).Columns[0].ModelParameters as NormalizingTransformer.AffineNormalizerModelParameters<float>;
```
This doesn't look like `data`.

Reply:
yeah its just a test and modified version of the above test but ok.

---
In reply to: [271538809](https://github.com/dotnet/machinelearning/pull/3172#discussion_r271538809) [](ancestors = 271538809)